src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,16 @@
 #include "BatteryStateOfChargeService.h"
+#include "LogFileReader.h"
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const double MILISECONDS_TO_HOURS = 2.77778e-7;
```

Comment:
Milliseconds is spelt incorrectly, we're engineers but we should get spelling right at least haha.